fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -191,6 +188,21 @@ struct MutationRef {
 		return mutation;
 	}
 
+	MutationRef decrypt(const std::unordered_map<BlobCipherDetails, Reference<BlobCipherKey>>& cipherKeys,
```

Comment:
Method seems to be duplicate of `CommitTransaction.h::decrypt` ? Any reason we can't use the same method here?

Reply:
We do use the other decrypt method inside this one (last line of this function). We have both because one takes in a map while the other takes a single text and header cipher key. If we remove this method there's a bunch of places that need to change